$NetBSD: patch-configure,v 1.1 2012/08/09 10:21:19 jperkin Exp $

Require _XOPEN_SOURCE=600 on Solaris for sockaddr_in6 and sockaddr_storage.

--- configure.orig	Thu Aug  9 09:32:17 2012
+++ configure	Thu Aug  9 09:36:08 2012
@@ -21706,7 +21706,7 @@
         # Solaris' C library apparently needs these runes to be threadsafe...
         CFLAGS="$CFLAGS -D_POSIX_PTHREAD_SEMANTICS -D_REENTRANT"
         # ... and this opt-in to get file descriptor passing support
-        CFLAGS="$CFLAGS -D_XOPEN_SOURCE=500"
+        CFLAGS="$CFLAGS -D_XOPEN_SOURCE=500 -D__EXTENSIONS__"
         ;;
 esac
 
